test/unit/builtins/numbers.spec.ts

```diff
@@ -48,6 +48,18 @@ describe("Number", () => {
     });
 });
 
+const toStringRadixes = [undefined, 10, 2, 8, 9, 16, 17, 36, 36.9];
+const toStringValues = [...numberCases, 1024, 1.2, NaN];
+// TODO: flatMap
+const toStringPairs = toStringValues.reduce<Array<readonly [number, number | undefined]>>(
```

Comment:
This is very hard to read, maybe we could make some helper function `combinations` or `testCombinations`. Also do we really need to check `NaN` for every radix value? Might be nicer as a separate test.